.env_SAMPLE

```diff
@@ -91,6 +91,7 @@ export ES_PORT=9200
 export ES_HOST=elasticsearch2
 export ES7_HOST=elasticsearch
 export SHEER_ELASTICSEARCH_INDEX=content
+#export ES_INDEX_PREFIX=local-
```

Comment:
Any reason not to just make this on by default?

Reply:
I recalled us talking about using no prefix for local and production, so I excluded it by default but wanted to leave the comment to indicate how it could be set. We can use it locally, it doesn't change the interaction at all codewise. Personally I like local matching prod so I can copy the curl commands from kibana without adjusting the index but thats negligible gain.

Reply:
Makes sense :+1: Leave as-is.